app/src/main/java/org/kiwix/kiwixmobile/nav/destination/reader/ReaderFragment.kt

```diff
@@ -87,8 +87,8 @@ class ReaderFragment : CoreReaderFragment() {
     }
   }
 
-  override fun showHomePage() {
-    exitBook()
+  override fun addBooks(books: MutableList<BooksOnDiskListItem>?) {
```

Comment:
this method does nothing in both subclasses of CoreReaderFragment so it is definitely useless, keep puling the thread